test/activityRepository-test.js

```diff
@@ -17,7 +17,7 @@ describe('ActivityRepository', () => {
   beforeEach(() => {
     today = '2020/08/22';
     anotherDay = '2020/08/21';
-    activityTestData = dummyActivityData ;
+    activityTestData = dummyActivityData;
```

Comment:
I am curious when/how to switch from our dummy data to the suppplied data with the project.